.github/workflows/linux-wpt.yml

```diff
@@ -48,25 +48,13 @@ jobs:
         with:
           name: release-binary
           path: release-binary
-      - name: Cache libffi
-        id: cache
-        uses: actions/cache@v3
-        with:
-          path: ./libffi6_3.2.1-8_amd64.deb
-          key: cache-libffi
-      - name: Download libffi
-        if: steps.cache.outputs.cache-hit != 'true'
-        run: |
-          wget http://mirrors.kernel.org/ubuntu/pool/main/libf/libffi/libffi6_3.2.1-8_amd64.deb
       - name: unPackage binary
         run: tar -xzf release-binary/target.tar.gz
       - name: Prep test environment
         run: |
           python3 -m pip install --upgrade pip
           sudo apt update
-          sudo apt install -qy --no-install-recommends libgl1 libssl1.1 libdbus-1-3 libxcb-xfixes0-dev libxcb-shape0-dev libunwind8 libgl1-mesa-dri mesa-vulkan-drivers libegl1-mesa
```

Comment:
I am not 100% sure that all things for webgpu are installed via `mach bootstrap`. It would be nice if you could dispatch workflow with production profile, disabled unit-tests and wpt string: `_webgpu`, to confirm that everything is still working.

Reply:
If there are more requirements for WebGPU these should move to `./mach bootstrap`.

Reply:
Unless, I guess, those requirements are hardware-dependent...is that the case here?

Reply:
I've updated the branch to install the one deb that wasn't installed by `./mach bootstrap`.